UPGRADE-4.0.md

```diff
@@ -228,6 +228,8 @@ Form
    ));
    ```
 
+ * Removed `ChoiceLoaderInterface` implementation in `TimezoneType`.
```

Comment:
`Removed `ChoiceLoaderInterface` implementation in `TimezoneType`. Use the "choice_loader" option instead.`

?